src/callbacks.js

```diff
@@ -78,6 +78,14 @@
  * @param {number} view - Type of view. Can be pc.VIEW_CENTER, pc.VIEW_LEFT or pc.VIEW_RIGHT. Left and right are only used in stereo rendering.
  */
 
+ /**
+ * @callback pc.callbacks.CalculateSortDistance
```

Comment:
Not sure callback property here is required, as there is no callback function as parameter in your code anywhere.

Reply:
It's there for the Typescript bindings, I think.

Reply:
@Maksims all the other callbacks seemed to be listed in this manner (see [the full file](https://github.com/playcanvas/engine/blob/3ab4274269f6e1a4cbc5c1c2b941bb30b2509db2/src/callbacks.js)) so I assumed that it was useful to add the new callback as well.